runtime/onert/frontend/nnapi/wrapper/OperationFactory.cc

```diff
@@ -2079,6 +2079,24 @@ OperationFactory::OperationFactory()
   };
 }
 
+_map[ANEURALNETWORKS_FUSEBATCHNORM_EX] = [](const OperationFactory::Param &init_param, Operands &) {
+    assert(init_param.input_count == 3 && init_param.output_count == 1);
```

Comment:
`assert` condition does not meet below comment.

Reply:
I will modify